include/upipe/upipe_helper_uclock.h

```diff
@@ -160,6 +160,16 @@ static void STRUCTURE##_clean_uclock(struct upipe *upipe)                   \
     uclock_release(s->UCLOCK);                                              \
     /* If the request was registered, it should be unregistered             \
      * automatically. Otherwise it has not been initialized. */             \
+}                                                                           \
+/** @internal @This returns the current system time.                        \
+ *                                                                          \
+ * @param upipe description structure of the pipe                           \
+ * @return the current systemt time                                         \
```

Comment:
Typo "systemt". Also, it might be useful to describe what is this UINT64_MAX return value.